src/jbuf/jbuf.c

```diff
@@ -702,26 +732,6 @@ uint32_t jbuf_packets(const struct jbuf *jb)
 }
 
 
-/**
- * Get number of current frames
- *
- * @param jb Jitter buffer
- *
- * @return number of frames
- */
-uint32_t jbuf_frames(const struct jbuf *jb)
```

Comment:
Why is this dropped? It's useful for unit testing and this ensures frame counting works correct.

Reply:
okay, I'll add it.